docs/reference/ingest/processors/inference.asciidoc

```diff
@@ -735,3 +735,69 @@ You can also specify the target field as follows:
 
 In this case, {feat-imp} is exposed in the
 `my_field.foo.feature_importance` field.
+
+
+[discrete]
+[[inference-processor-examples]]
+==== {infer-cap} processor examples
+
+The following example uses an <<inference-apis,{infer} endpoint>> in an {infer} processor named `query_helper_pipeline` to perform a chat completion task.
+The processor generates an {es} query from natural language input using a prompt designed for a completion task type.
```

Comment:
It would be helpful to link to docs showing how to setup an Inference API Endpoint for the `chat` completion task.

Reply:
I updated via [2b7e8cc](https://github.com/elastic/elasticsearch/pull/116018/commits/2b7e8cc3ec3142821318cf7725108480f859678a).